examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,24 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
+    imgs = image.iter_img(msdl_atlas_dataset.maps)
+    regions_coords = np.array([
+        map(np.asscalar, plotting.find_xyz_cut_coords(img)) for img in imgs])
+    np.random.seed(42)
+    node_colors = np.random.rand(39, 3)
```

Comment:
Instead of `39`, better to use `len(regions_coords)` (I'm using this function, but with a modified version of the maps image :) )